frontend/lib/src/theme/getColors.ts

```diff
@@ -83,7 +83,7 @@ export const createEmotionColors = (genericColors: {
     ...genericColors,
     ...derivedColors,
 
-    codeTextColor: genericColors.green80,
+    codeTextColor: genericColors.green,
```

Comment:
green and green80 is the same, but `green` is what we define in our theme as the main green color